tree_unique_args/src/loop_invariant.egg

```diff
@@ -0,0 +1,28 @@
+;; Loop Invariant
+
+;; bool: whether the term in the Expr is an invariant.
+(function is-inv-Expr (IdSort Expr) bool :unextractable :merge (or old new))
+(function is-inv-ListExpr (IdSort ListExpr) bool :unextractable :merge (or old new))
+
+(relation arg-inv (IdSort i64))
```

Comment:
Why do we need `arg-inv` and `is-inv-Expr`?
Isn't `(arg-inv id i)` equivalent to `(is-inv-Expr id (Get (Arg id) i))`

Reply:
True, it was used before changing to BodyContains for trigering find-invariant, but now it's not needed

Reply:
fixed